src/elements/FunctionsPanelParameters/functionsPanelParameters.util.js

```diff
@@ -0,0 +1,85 @@
+import { isNil } from 'lodash'
+import { stringToBoolean } from '../../utils/stringToBoolean'
+
+export const STRING_TYPE = 'string'
+export const NUMBER_TYPE = 'number'
+export const BOOLEAN_TYPE = 'boolean'
+export const JSON_TYPE = 'json'
+
+export const parameterTypeOptions = [
+  { label: 'String', id: STRING_TYPE },
+  { label: 'Number', id: NUMBER_TYPE },
+  { label: 'Boolean', id: BOOLEAN_TYPE },
+  { label: 'JSON', id: JSON_TYPE }
+]
+
+export const tableHeaders = [
+  { label: 'Name', id: 'name' },
+  { label: 'Type', id: 'type' },
+  { label: 'Value', id: 'value' }
+]
+
+export const isNameNotUnique = (name, parameters) => {
+  return parameters.some(route => route.data.name === name)
+}
+
+export const isParameterValid = parameter => {
+  return parameter.name.length > 0 && String(parameter.value).length > 0
```

Comment:
Please take into consideration that `String(undefined)` is not `''` but `'undefined'`.
So if `parameter.value` is `undefined`, `String(parameter.value)` will be `undefined`, and its `.length` greater than 0.